`kubernetes_cluster/node_pool`- `capacity_reservation_group_id` (#17395)
hashicorp/terraform-provider-azurerm: website/docs/r/kubernetes_cluster_node_pool.html.markdown

---

> * `capacity_reservation_group_id` - (Optional) AKS will associate the specified agent pool with the Capacity Reservation Group. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `capacity_reservation_group_id` - (Optional) Specifies the ID of the Capacity Reservation Group where this Node Pool should exist. Changing this forces a new resource to be created.
```
